includes/definitions/discovery/fs-nmu.yaml

```diff
@@ -1,5 +1,42 @@
+mib: OAP-NMU
 modules:
     os:
-        hardware: OAP-C1-OEO::vDeviceType.0
-        serial: OAP-C1-OEO::vSerialNumber.0
```

Comment:
If you look at the failing tests you see that this causes the information to be missing, you can add multiple oids instead. Any reason you removed it?

Reply:
The OEO is a module in an FS NMU device. The vDeviceType and vSerialNumber will  based on which OEO is installed or replaced in position C1, if an OEO Module is even installed at position C1. In place of the OEO Module, there might be an EDFA Module, or a MUX | DEMUX Module (which has nothing to SNMP), or another FS Module installed at position C1.

For the FS-NMU devices hardware, serial, and version, the above was removed and replaced with the OAP-NMU::serialNumber.0, OAP-NMU::softwareVersion.0 and OAP-NMU::hardwareVersion.0 to report the details of the FS NMU device rather than a module installed within it.

Reply:
modified fs-nmu test data